test/core/userdata/0.8/packages/MeshToolkit/extra/backup/ConvertMeshIntoLinePointsSurfaces.dyn.0.backup

```diff
@@ -0,0 +1,46 @@
+<Workspace Version="0.8.2.1957" X="26125.1497935841" Y="22672.9170313985" zoom="1.15636491448696" Name="Home" RunType="Manual" RunPeriod="1000" HasRunWithoutCrash="True">
```

Comment:
Is `backup` folder created by you?

Reply:
My bad. That seems to be created automatically when I run the tests. I'll delete that from the PR